app/_data/docs_nav_ee_0.35-x.yml

```
@@ -0,0 +1,210 @@
+- title: Deployment
+  items:
+    - text: Licensing
+      items:
+        - text: Overview
+          url: /deployment/licensing
+        - text: How to Access Your License
+          url: /deployment/access-license
+    - text: Installation
+      items:
+        - text: Docker
+          url: /deployment/installation/docker
+        - text: CentOS
+          url: /deployment/installation/centos
+        - text: Amazon Linux
+          url: /deployment/installation/amazon-linux
+    - text: Changelog
+      url: /enterprise/changelog
+      absolute_url: true
+    - text: Migrating to 0.35
+      url: /deployment/migrations
+
+- title: Getting Started
+  url: /getting-started/
+  items:
+    - text: Overview
+      url: /getting-started/
+    - text: How to Start Kong Enterprise
+      url: /getting-started/start-kong
+    - text: How to Add a Workspace
+      url: /getting-started/add-workspace
+    - text: How to Add a Role and Permissions
+      url: /getting-started/add-role
+    - text: How to Add an Admin
+      url: /getting-started/add-admin
+    - text: How to Add a Service and Route
+      url: /getting-started/add-service
+    - text: How to Enable a Plugin
+      url: /getting-started/enable-plugin
+    - text: How to Add a Consumer
+      url: /getting-started/add-consumer
+    - text: How to Enable the Dev Portal
+      url: /getting-started/enable-dev-portal
+
+- title: Admin API
+  url: /admin-api/
+  items:
+  - text: Workspaces
+    url: /admin-api/workspaces/reference
+  - text: RBAC
+    url: /admin-api/rbac/reference
+  - text: Admins
+    url: /admin-api/admins/reference
+    items:
+      - text: API Reference
+        url: /admin-api/admins/reference
+      - text: Examples
+        url: /admin-api/admins/examples
+  - text: Vitals
+    url: /admin-api/vitals
+    items:
+      - text: Vitals Overview
+        url: /admin-api/vitals
+      - text: Kong Vitals with InfluxDB
+        url: /admin-api/vitals/vitals-influx-strategy
+      - text: Kong Vitals with Prometheus
+        url: /admin-api/vitals/vitals-prometheus-strategy
+  - text: Audit Logging
+    url: /admin-api/audit-log
+
+- title: Kong Manager
+  items:
+    - text: Overview
+      url: /kong-manager/overview
+    - text: Networking
+      url: /kong-manager/networking/configuration
+      items:
+        - text: Default and Custom Configuration
+          url: /kong-manager/networking/configuration
+        - text: Configuring Kong Manager to Send Email
```

Still wraps on the sidenav

```suggestion
        - text: Configuring to Send Email
```